test/helpers/application_helper_test.rb

```diff
@@ -114,4 +114,18 @@ def test_generate_link_for
         '<a class="btn" href="javascript:foo()">link</a>')
     end
   end
+
+  describe 'host details ui path generation' do
+    setup do
+      Setting[:host_details_ui] = true
+    end
+
+    test 'current_host_details_path should fallback to host id when name is removed during edit' do
```

Comment:
I'd suggest to write test description of what it must do, not what it should to.
```suggestion
    test 'current_host_details_path falls back to host id for host without a name' do
```

Reply:
updated the description